gui/src/mainwindow.cpp

```diff
@@ -768,6 +773,7 @@ void MainWindow::updateFromUI()
     pins[PIN_PPMIN] = ppin_index==1?-1:ppin_index;
     int but_index = ui->cmbButtonPin->currentIndex()+1;
     pins[PIN_BUTRESET] = but_index==1?-1:but_index;
+    // TODO something to do here for ALERT_CH?
```

Comment:
similarly here -- do I need to do similar treatment for alertch behavior?

Reply:
No nothing needed here. I only bother checking in the GUI/FW if the user selected two physical output pins that are the same. For the output channels you can select the same channel so whatever item is last in sense.cpp will be set. 
If I have alert=CH1 and tilt=CH1 will happily let you set this. Which is also why we should keep Tilt roll and pan as the last set item in the list.

Reply:
got it -- updated sense.cpp